scripts/findBadKeys.js

```diff
@@ -48,6 +48,10 @@ function* iterateComponentFiles() {
 const checkPathVsKey = () => {
   const iterator = iterateComponentFiles();
   for (const file of iterator) {
+    if (file.split("/").pop() === "test-event.mjs") {
```

Comment:
Awesome!
Can you put this inside a new function `isTestEvent()` like [this one](https://github.com/PipedreamHQ/pipedream/blob/master/scripts/findBadKeys.js#L16) and make it skip alongside with [these conditions](https://github.com/PipedreamHQ/pipedream/blob/master/scripts/findBadKeys.js#L42)?